components/partials/portfolio/bridge/token-selector/select.vue

```diff
@@ -34,44 +33,70 @@
-                placeholder="0.0000"
-                error-classes="mt-4"
-                :errors="errors"
-                :valid="valid"
-                :max="balanceToFixed"
-                :max-decimals="value.decimals"
-                :max-selector="balance.gt(0.0001)"
-                :value="amount"
-                data-cy="token-selector-amount-input"
-                @input="handleAmountChange"
-                @blur="resetIsSearching"
-                @mousedown.native.stop="focusInput"
-              />
-              <img
-                v-if="logo"
-                :src="logo"
-                :alt="name"
-                class="rounded-full w-6 h-6"
-              />
-              <v-icon-category-alt v-else class="rounded-full w-6 h-6" />
-              <span
-                class="font-bold text-lg pl-2 pr-3 text-gray-200 tracking-wide break-normal"
-                data-cy="token-selector-selected-text-content"
-              >
-                {{ symbol }}
-              </span>
+            <div class="flex flex-col">
+              <div class="flex items-center w-full">
+                <v-input
+                  id="bridge-input"
+                  dense
+                  :small="small"
+                  :lg="lg"
+                  transparent-bg
+                  type="number"
+                  step="0.01"
+                  min="0"
+                  placeholder="0.0000"
+                  :errors="errors"
+                  hide-errors
+                  :valid="valid"
+                  :max="balanceToFixed"
+                  :max-decimals="value.decimals"
+                  :max-selector="!disableMaxSelector && balance.gt(0.0001)"
+                  :max-classes="'input-max-button'"
+                  :value="amount"
+                  :prefix="prefix"
+                  :input-classes="prefix ? 'pl-0' : ''"
+                  data-cy="token-selector-amount-input"
+                  @input="handleAmountChange"
+                  @input-max="handleMax"
+                  @blur="resetIsSearching"
+                  @mousedown.native.stop="focusInput"
+                />
+                <img
+                  v-if="logo"
+                  :src="logo"
+                  :alt="name"
+                  class="rounded-full w-4 h-4"
+                />
+                <v-icon-category-alt v-else class="rounded-full w-4 h-4" />
+                <span
+                  class="font-bold text-lg px-3 text-gray-200 tracking-wide break-normal"
+                  data-cy="token-selector-selected-text-content"
+                >
+                  {{ symbol }}
+                </span>
+                <div class="block pr-4 text-white">
+                  <v-icon-caret-down-slim />
+                </div>
+              </div>
+              <div class="flex items-center justify-between w-full px-4 mt-1">
+              <span v-if="errors.length > 0" class="text-red-400 text-[12px]">{{ formatError(errors) }}</span>
```

Comment:
Do we need the remove `formatError` function, since we've added the `v-if="errors.length > 0"` check in the html template, can we do `error[0]` instead?

Reply:
Yes, we can. I think I added the ```v-if="errors.length > 0"``` check later.